### PR TITLE
Feature/m2 305 create activity schedule notification tab

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

File: src/features/Schedule/CreateActivityPopup/AvailabilityTab/AvailabilityTab.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    setValue('timeout.access', false);
+  };
+
+  return (
+    <>
+      <SelectController
+        name="availability"
+        fullWidth
+        options={availabilityOptions}
+        label=""
```
Comment (wildkiomi) [inline src/features/Schedule/CreateActivityPopup/AvailabilityTab/AvailabilityTab.tsx:45]: can you remove this empty prop?

File: src/components/FormComponents/InputController/InputController.styles.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+export const StyledTextField = styled(TextField)`
+  input[type='number']::-webkit-inner-spin-button,
+  input[type='number']::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+`;
+
+export const StyledAdornment = styled(Box)`
```
Comment (nmialik) [inline src/components/FormComponents/InputController/InputController.styles.ts:13]: use can use StyledFlexTopCenter

File: src/resources/app-fr.json [hunk 3]
```diff
@@ -169,6 +169,23 @@
   "selectEndingDate": "Sélectionnez la date de fin",
   "date": "Date",
   "startingEndingDate": "Date de début/fin",
+  "activity*": "Activity*",
```
Comment (nmialik) [inline src/resources/app-fr.json:172]: is this translation used somewhere in the code? if yes then use pls `required`

File: src/features/Schedule/CreateActivityPopup/NotificationsTab/Reminder/Reminder.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        <StyledFlexTopCenter>
+          <StyledInputWrapper>
+            <InputController
+              label={t('activityIncomplete')}
+              type="number"
+              name="reminder.activityIncomplete"
+              control={control}
+              InputProps={{ inputProps: { min: 1 } }}
+              endTextAdornmentSingular={t('day')}
+              endTextAdornmentPlural={t('days')}
```
Comment (nmialik) [inline src/features/Schedule/CreateActivityPopup/NotificationsTab/Reminder/Reminder.tsx:39]: I suppose we can use this for plural - https://www.i18next.com/translation-function/plurals#singular-plural

Comment (nmialik) [reply src/features/Schedule/CreateActivityPopup/NotificationsTab/Reminder/Reminder.tsx:39]: you just need to call `t('day', {count: {countDays}})`
and create two translations: 
```
"day_one": "day",
"day_other": "days"
```

**Review by nmialik — APPROVED** [review]

**Pull request merged**